objects.cpp

```diff
@@ -55,14 +55,23 @@ class Tangible : public Stocked {
 };
 
 class Cart {
+    static int requests;
```

Comment:
@MCJOHN974 why making it static instead of a global variable, which we already have?